### DISCUSSION TITLE
a way to see all duplicates of all location

### DISCUSSION THREAD

**Discussion by tampadesignr** [spacedriveapp/spacedrive#2776] [Feature Request]
a way to see all duplicates of all location
i posted this before i found this
[https://www.reddit.com/r/software/comments/1g63ixw/software_to_sort_media_by_keywords_not_by_ext/](url)
however i dont think it does what i need it to do
https://www.spacedrive.com/docs/product/guides/duplicates
doesnt say how to accomplish what i would like to do
basically a result page of all found duplicates from everything it indexed
similar to
https://github.com/0x90d/videoduplicatefinder
that one starts to bog down and take forever and crash with large amounts of info

**Comment by HeavenVolkoff** [comment]
Thanks for the suggestions. This feature is planned for beta
